cmd/ignite-spawn/ignite-spawn.go

```diff
@@ -56,11 +57,6 @@ func StartVM(vm *api.VM) error {
 	metricsSocket := path.Join(vm.ObjectPath(), constants.PROMETHEUS_SOCKET)
 	serveMetrics(metricsSocket)
```

Comment:
Maybe use a waitgroup or something here to actually unblock the goroutine later when we know the firecracker process has started?

Reply:
We can only get close to the startup place, as the Firecracker call is blocking. `ignite-spawn` cannot actually track Firecracker's state (without listening to some sockets), so this cannot be done at the moment.

Reply:
Ok, good for now